### PR TITLE
op-guide: update monitor config url

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @tennix @shenli

### CODE REVIEW

File: op-guide/monitoring-tidb.md [hunk 1]
```diff
@@ -233,7 +233,7 @@ Add the Push Gateway address to the yaml configuration file:
 
 2. On the sidebar menu, click "Dashboards" -> "Import" to open the "Import Dashboard" window.
 
-3. Click "Upload .json File" to upload a JSON file ( Download [TiDB Grafana Config](https://github.com/pingcap/docs/tree/master/etc) ). 
+3. Click "Upload .json File" to upload a JSON file ( Download [TiDB Grafana Config](https://grafana.com/users/tennix) ).
```
Comment (shenli) [inline op-guide/monitoring-tidb.md]: The url is not correct.

Comment (nolouch) [inline op-guide/monitoring-tidb.md]: I think this place is more easy to maintain

Comment (siddontang) [inline op-guide/monitoring-tidb.md]: can we create a pingcap user in grafana?

Comment (tennix) [inline op-guide/monitoring-tidb.md]: yes, we can create an organization on Grafana sites, or I can change my username to an organization

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@tennix PTAL

**Comment by tennix** [comment]
LGTM

**Pull request merged**